frontend/src/modules/entitieslist/components/EntitiesList.js

```diff
@@ -329,6 +329,7 @@ export class EntitiesListBase extends React.Component<InternalProps> {
     render() {
         const props = this.props;
         const search = props.parameters.search;
```

Comment:
Now that you pass all parameters as props, it doesn't make much sense to pass the search parameter separately. So you can remove this line and the prop from the `Entity` component, where you'll need to declare search from parameters.